oe-account-service/op-energy-account-service/src/OpEnergy/BlockTimeStrike/Server/V1/BlockTimeStrikeGuessService.hs

```diff
@@ -355,18 +387,17 @@ getBlockTimeStrikesGuessesPage mpage mfilter = profile "getBlockTimeStrikesGuess
             )
          )
       .| ( C.awaitForever $ \(guess, strikeE@(Entity strikeId _)) -> do
-           mObserved <- lift $ selectFirst [ BlockTimeStrikeObservedStrike ==. strikeId ][]
```

Comment:
This looks like the most important diff to understand, in terms of the effect of the current work. 

I am trouble following it though.